### PR TITLE
fix: update installed crds programmatically

### PR DESCRIPTION
**Pull request by boranx**
Additional fix: https://issues.redhat.com/browse/OSD-7598

### CODE REVIEW

**Review by fahlmant — APPROVED** [review]
/lgtm

**Comment by cblecker** [comment]
/lgtm cancel
/hold

We need to update https://github.com/openshift/managed-velero-operator/blob/adc498a35c74a4ba2a46a74cc04bac57907d83d3/pkg/controller/velero/velero.go#L39-L41 to the latest versions too

**Comment by boranx** [comment]
@cblecker nice catch, thanks! It's now squashed into 1 commit

File: pkg/velero/crds.go [hunk 1]
```diff
@@ -19,16 +19,15 @@ func InstallVeleroCRDs(log logr.Logger, client client.Client) error {
...
+	for _, unstructuredCrd := range veleroInstall.AllCRDs("v1").Items {
+		// Get upstream crds
+		crd := &apiv1.CustomResourceDefinition{}
 		if err := runtime.DefaultUnstructuredConverter.FromUnstructured(unstructuredCrd.Object, crd); err != nil {
 			return err
 		}
-		// Add Conversion to the spec, as this will be returned in the founcCrd
-		crd.Spec.Conversion = &apiextv1beta1.CustomResourceConversion{
-			Strategy: apiextv1beta1.NoneConverter,
-		}
```
Comment (cblecker) [inline pkg/velero/crds.go:31]: This shouldn't have been removed. Opening a PR to re-add.

**Pull request merged**